.cirrus.yml

```diff
@@ -0,0 +1,56 @@
+env:
+  MAVEN_OPTS: -Xmx512M -XX:+ExitOnOutOfMemoryError
+  MAVEN_SKIP_CHECKS_AND_DOCS: -Dair.check.skip-all=true -Dmaven.javadoc.skip=true
+  MAVEN_FAST_INSTALL: -DskipTests $MAVEN_SKIP_CHECKS_AND_DOCS -B -q -T 1C
+
+webui_checks_task:
+  container:
+    image: node:latest
+  check_script: presto-main/bin/check_webui.sh
+
+maven_checks_task:
+  container:
+    image: maven:3-jdk-8
+  maven_cache:
+    folder: ~/.m2
+  test_script: mvn install -DskipTests -B -T 1C -P ci
+  docker_script: docker/build-local.sh
+
+maven_specific_modules_task:
+  container:
+    image: maven:3-jdk-8
+  maven_cache:
+    folder: ~/.m2
+  env:
+    matrix:
```

Comment:
is it "enumerate" or "multiple"? (71496aa777ef036896617f1943fb4b0c86932338)

Reply:
See https://cirrus-ci.org/guide/writing-tasks/#matrix-modification